### ISSUE TITLE
Accidental bump of required version of i3ipc?

### ISSUE THREAD

**Issue by dburian** [fennerm/flashfocus#46]
When trying to install flashfocus through pip3, the following error was thrown:
```
Could not find a version that satisfies the requirement i3ipc<3.0,>=2.1.2 (from flashfocus) (from versions: 1.0.1, ..., 2.0.1, 2.1.1)
No matching distribution found for i3ipc<3.0,>=2.1.2 (from flashfocus)
```
After some digging around I found that i3ipc newest version is 2.1.1. I could be wrong but it seems like in the last commit 552ecb421c4db24d24edc758a53de58c0d01406d you may have accidentally bumped not only flashfocus version, but also version of i3ipc (both from 2.1.1 to 2.1.2)?
Again, could be wrong so apologies if I overlooked something...

**Comment by fennerm** [comment]
Oh boy I suspect you’re right, thanks for reporting. I’ll make a patch release later today.

**Comment by fennerm** [comment]
Alright should be fixed in v2.1.3. Thanks again!
